projects/plugins/jetpack/modules/widgets/social-icons.php

```diff
@@ -52,6 +56,19 @@ public function __construct() {
 			add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_icon_scripts' ) );
 			add_action( 'wp_footer', array( $this, 'include_svg_icons' ), 9999 );
 		}
+
+		// add_filter( 'widget_types_to_hide_from_legacy_widget_block', array( $this, 'hide_widget_in_block_editor' ) );
```

Comment:
I'm seeing the Social Icons option still appear in the Legacy Widget for me. Is this line supposed to be un-commented?

Reply:
Oh haha nice catch. I had that turned off while testing.

Reply:
Updated to 'chain' instead od 'default' and removed the comment from the hook. 👍